cmd/dns_remove.go

```diff
@@ -54,11 +54,7 @@ func removeDomainRecord(domainIdent, recordIdent string, force bool) error {
 		}
 
 		_, err = globalstate.EgoscaleV3Client.Wait(ctx, op, v3.OperationStateSuccess)
-		if err != nil {
-			return fmt.Errorf("exoscale: error while waiting DNS record deletion: %w", err)
```

Comment:
You're right that these can be removed, but I like these error messages. Makes it easier to find out where an error happened when you read the logs. IMHO these can be kept :)

Reply:
@mlec1 What I wanted to point out here in this PR, if you were not wrapping/modifying the error, it was not needed to add an extra condition and return it directly instead.
But in the case you want to add more context on the error like here, that is fine 👍 you can keep it.

I have no strong opinion here, but if you decided to add more context to the error here, looks good to me.